main.go

```diff
@@ -36,6 +36,7 @@ func init() {
 	flag.BoolVar(&f.ReplicateRoles, "replicate-roles", true, "Enable replication of roles")
 	flag.BoolVar(&f.ReplicateRoleBindings, "replicate-role-bindings", true, "Enable replication of role bindings")
 	flag.BoolVar(&f.ReplicateServiceAccounts, "replicate-service-accounts", true, "Enable replication of service accounts")
+	flag.BoolVar(&f.SyncByContent, "sync-by-content", false, "Always compare the contents of source and target resources and force them to be the same")
```

Comment:
Yes, I've added a section to the README. Please have a look. Maybe it would be better to have a section explaining all command line options. But that could be another PR.

Reply:
True, I'd agree that a major restructuring of the README would be out-of-scope for this PR. I'll go ahead and merge this one. 🙂👍